doc/user/content/sql/drop-table.md

```diff
@@ -19,30 +19,68 @@ statements. `DROP TABLE` removes tables from your Materialize instance.
 
 Field | Use
 ------|-----
-**IF EXISTS** | Do not return an error if the named table does not exist.
-_table&lowbar;name_ | The name of the table to drop.
-
+_data_type_name_ | The name of the table to remove.
+`CASCADE` | Automatically removes any objects that depend on the table, as well as the table.
+`IF EXISTS`  | Do not issue an error if the named table doesn't exist.
+`RESTRICT`  | Don't remove the table if any objects depend on it. _(Default.)_
```

Comment:
this should be "any non-index objects"; we'll drop the table and its indexes even under `RESTRICT`.